Cart_Reader/Config.h

```diff
@@ -124,7 +124,7 @@
 
 /****/
 
-/* [ Flashrom Programmer for SNES repros -------------------------- ]
+/* [ Flashrom Programmer for repro carts -------------------------- ]

```

Comment:
Since enabling this option actually turns on repro carts for not just SNES but MD, N64 and now PCE.